PxtlCa.XmlCommentMarkDownGenerator/TagRenderers.cs

```diff
@@ -25,89 +22,97 @@ public TagRenderer(string formatString, Func<XElement, string, IEnumerable<strin
 
         public static Dictionary<string, TagRenderer> Dict { get; } = new Dictionary<String, TagRenderer>()
         {
-            {"doc", new TagRenderer(
+            ["doc"] = new TagRenderer(
```

Comment:
Ah, so *that's* the new dictionary immediate syntax.  I knew it was a thing but hadnt' seen it yet.